python/hsml/model.py

```diff
@@ -291,7 +296,27 @@ def get_feature_view(self):
             `Exception` in case the backend fails to retrieve the tags.
         """
         fv_prov = self.get_feature_view_provenance()
-        return explicit_provenance.Links.get_one_accessible_parent(fv_prov)
+        fv = explicit_provenance.Links.get_one_accessible_parent(fv_prov)
+        if fv is None:
+            return None
+        if init:
+            td_prov = self.get_training_dataset_provenance()
+            td = explicit_provenance.Links.get_one_accessible_parent(td_prov)
+            is_deployment = "DEPLOYMENT_NAME" in os.environ
+            if online or is_deployment:
+                _logger.info(
+                    "Initializing serving"
+                    + (" - deployment detected" if is_deployment else "")
+                )
+                fv.init_serving(training_dataset_version=td.version)
```

Comment:
What about the rest of arguments for [init_serving](https://docs.hopsworks.ai/feature-store-api/latest/generated/api/feature_view_api/#init_serving)? Should we accept a `init_serving_config` param in the `get_feature_view` method of type dict and pass it to the init_serving method?